beetmoverscript/src/beetmoverscript/task.py

```diff
@@ -83,14 +84,28 @@ def validate_bucket_paths(bucket, s3_bucket_path):
         raise ScriptWorkerTaskException("Forbidden S3 {} destination".format(s3_bucket_path))
 
 
-def check_maven_artifact_map(context):
-    """Check that versions in artifact map are valid Maven versions"""
+def get_maven_version(context):
+    """Extract and validate a valid Maven version"""
     version = context.task["payload"]["version"]
+    release_props = context.release_props
+    # Version follows the FirefoxVersion pattern for Geckoview < 84.0
```

Comment:
Nit: Add a TODO section to remind us to remove that code path once Geckoview 84 is in mozilla-release